subprojects/platform-native/src/main/java/org/gradle/nativeplatform/toolchain/internal/xcode/AbstractLocator.java

```diff
@@ -41,6 +41,11 @@ public File find() {
                 execAction.executable("xcrun");
                 execAction.workingDir(System.getProperty("user.dir"));
                 execAction.args(getXcrunFlags());
+
+                String developerDir = System.getenv("DEVELOPER_DIR");
+                if (developerDir != null) {
+                    execAction.environment("DEVELOPER_DIR", developerDir);
+                }
```

Comment:
I think we inherit the environment from the parent process, is this necessary?

Reply:
We may, I wanted to be explicit so we know where to wire things properly when we support the feature on the tool chain. Do you think we should remove as much as we can?